src/modules/doubt/schema/doubtAnswer.schema.ts

```diff
@@ -0,0 +1,27 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import * as mongoose from 'mongoose';
```

Comment:
Document

Reply:
yeah i've fixed them in the original PR #108 
The present PR#109 is just a continuation for documentation, and all changes of PR#108 will be merged in that later by me